src/platform/testing/e2e/nightwatch-assertions/isAllyDeepEquals.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable valid-typeof */
+
+/**
+ * Checks if the number of focusable and tabbable elements are correct.
+ * This is a simple equality check, to make sure the right number of
+ * elements can be reached by pressing TAB or through programmatic
+ * focus with the JS .focus() method.
+ *
+ * ```javascript
+ *  this.demoTest = function (client) {
+ *    client.assert.isAllyDeepEquals("body", true);
+ *  };
+ * ```
+ *
+ * @method isAllyDeepEqual
+ * @param {string} [selector] The selector (CSS / Xpath) used to locate the element
+ * @param {boolean} [balanceBool] True if page has no no tabIndex="-1"
+ * @api assertions
+ */
+exports.assertion = function isAllyDeepEquals(selector, balanceBool) {
```

Comment:
:thinking: Can you help me understand what this is supposed to be doing. I'm not following its intended purpose, I think.

Looking at the related command (`allyDeepEquals`), it looks like this asserts that the number of tabbable elements matches the number of focusable elements within the `selector`. Is that right?

It's also worth noting that even if there _are_ the same number of each, that doesn't mean they're the same _set_ of elements. Consider:
```
Tabbable elements:
[ a b c ]

Focusable elements:
[ a b d ]
```

Reply:
@cvalarida I'm going to remove this function. Its usefulness was less and less clear as I started to work on this feedback. I've updated the focusable and tabbable functions to remove the external library and be more expressive.